test/integration/fury-test.es6

```diff
@@ -27,14 +29,19 @@ const refractedApi = [
               'key': ['string', {}, {}, 'tag'],
               'value': ['string', {}, {}, null]
             }]
-          ]],
+          ]]],
           ['transition', {}, {}, [
-            ['httpTransaction', {title: 'Get a frob', description: 'Gets information about a single frob instance'}, {}, [
-              ['httpRequest', {}, {method: 'GET'}, null],
+            ['httpTransaction', {title: 'Get a frob'}, {}, [
+              ['httpRequest', {}, {method: 'GET'}, [
+                ['copy', {}, {}, 'Gets information about a single frob instance']
```

Comment:
Shouldn't this copy be for a `transition`? I don't know why it was in `httpTransaction` in the first place.